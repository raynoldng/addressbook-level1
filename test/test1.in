clear
add Adam Brown p/111111 e/adam@gmail.com
list
exit