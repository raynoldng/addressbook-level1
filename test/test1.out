|| ===================================================
|| ===================================================
|| AddessBook Level 1 - Version 1.0
|| Welcome to your Address Book!
|| ===================================================
|| Using default storage file : addressbook.txt
|| Enter command: || [Command entered:clear]
|| Address book has been cleared!
|| ===================================================
|| Enter command: || [Command entered:add Adam Brown p/111111 e/adam@gmail.com]
|| New person added: Adam Brown, Phone: 111111, Email: adam@gmail.com
|| ===================================================
|| Enter command: || [Command entered:list]
|| 	1. Adam Brown  Phone Number: 111111  Email: adam@gmail.com
|| 
|| 1 persons found!
|| ===================================================
|| Enter command: || [Command entered:exit]
|| Exiting Address Book... Good bye!
|| ===================================================
|| ===================================================
